include/zen/scene/scene.h

```diff
@@ -8,9 +8,15 @@
 struct zn_scene {
   struct zn_screen_layout* screen_layout;
 
+  struct zn_view* active_view;
```

Comment:
```suggestion
  struct zn_view* focus_view;
```

Do we have to use different term, `active` and `focus`?